var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -28,17 +28,28 @@ class Lammps(CMakePackage, CudaPackage, ROCmPackage):
     #   marked deprecated=True
     # * patch releases older than a stable release should be marked deprecated=True
     version("develop", branch="develop")
+    version(
+        "20230802",
+        sha256="48dc8b0b0583689e80ea2052275acbc3e3fce89707ac557e120db5564257f7df",
```

Comment:
Weird.  I'm not getting this sha256 from `spack checksum`.

Reply:
Now I can confirm this sha256 BUT not the next one.  I made sure I have this PR checked out too.